n2y/main.py

```diff
@@ -27,11 +28,6 @@ def main(raw_args, access_token):
         help="Level to set the root logging module to",
     )
 
-    # TODO: Add the ability to dump out a "schema" file that contains the schema
```

Comment:
I removed these since they seem kind of out of scope and a long way off, if we did ever want to add them.